deployments/demos/local-containers/README.md

```diff
@@ -0,0 +1,209 @@
+# OpenADKit Remote Control Architecture: Implementation Report and User Manual
+
+## 1. Introduction
+
+This document provides a comprehensive implementation guide and technical overview for the distributed architecture of the OpenADKit project. The core objective is to separate compute-intensive and lightweight components of an autonomous driving system, enabling deployment across different hardware. For example, the core Autoware software stack runs on a high-performance cloud server, while simulators and visualization tools operate on a local developer's laptop.
+
+To achieve this, we utilize [Zenoh](https://zenoh.io/) as a high-performance, low-latency communication protocol, paired with the `zenoh-bridge-ros2dds` tool to seamlessly connect two ROS 2 (Robot Operating System 2) environments isolated by Docker virtual networks. This manual covers architecture design, setup steps, system startup, and troubleshooting, providing complete operational guidance.
+
+## 2. Detailed Architecture Design
+
+### 2.1. Core Concept: Client-Server Model
+
+The system decouples a previously monolithic architecture into a client-server model:
+
+- **Server Side**: Includes components with high computational demands (especially CPU and GPU).
+  - `autoware`: Core perception, decision-making, and planning modules.
+  - `scenario_simulator`: Generates virtual traffic environments, obstacles, and vehicle dynamics, providing simulated sensor data for Autoware.
+  - **Deployment**: Typically on cloud virtual machines (e.g., AWS, GCP) or dedicated lab servers.
```

Comment:
Just as we mentioned above, it should run in the vehicle or a powerful simulation server.